src/Components/Notifications/NotificationsTabs.tsx

```diff
@@ -0,0 +1,66 @@
+import { BaseTab, BaseTabs, Box, Clickable, useTabs } from "@artsy/palette"
+import { Sticky, StickyProvider } from "Components/Sticky"
+import styled from "styled-components"
+
+const TABS_CONTAINER_HEIGHT = 60
+
+export interface NofiticationsTabsProps {
+  mode: "dropdown" | "page"
+  maxDropdownHeight?: string
+}
+
+export const NofiticationsTabs: React.FC<NofiticationsTabsProps> = ({
+  mode,
+  maxDropdownHeight,
+  children,
+}) => {
+  const { tabs, activeTab, activeTabIndex, handleClick, ref } = useTabs({
+    children,
+  })
+
+  const Tabs = (
+    <BaseTabs ref={ref as any} left={0} right={0} px={2} bg="white100">
+      {tabs.map((tab, i) => {
+        return (
+          <Clickable
+            key={i}
+            ref={tab.ref as any}
+            aria-selected={i === activeTabIndex}
+            onClick={handleClick(i)}
+            flex={1}
+          >
+            <NotificationBaseTab active={i === activeTabIndex} variant="sm">
+              <span>{tab.child.props.name}</span>
+            </NotificationBaseTab>
+          </Clickable>
+        )
+      })}
+    </BaseTabs>
+  )
```

Comment:
**suggestion(non-blocking):** How do you feel about moving this to its own component? 🤔

Reply:
I've been thinking about that too. But I didn't find any advantages that these changes would give us (the component is used in one place, it is necessary to duplicate the TS types for props)

Reply:
consider this as resolved then